packages/elasticsearch-plugin/src/build-elastic-body.ts

```diff
@@ -109,14 +109,13 @@ export function buildElasticBody(
         }
     }
 
-    const sortArray = [];
+    const sortArray: ElasticSearchSortInput = [];
     if (sort) {
         if (sort.name) {
             sortArray.push({
                 'productName.keyword': { order: sort.name === SortOrder.ASC ? 'asc' : 'desc' },
             });
-        }
-        if (sort.price) {
+        } else if (sort.price) {
```

Comment:
This is a change in behaviour - now sorting by name/price is an exclusive OR. Was there a particular reason for this change? Does it not make sense to allow sorting on name _and_ price?

Reply:
Oh, no! This was a left-over from a first tryout!
Awesome that you saw this! :)